Adjust helpIcon styles

johnb254 opened the pull request:
The help icon was not easily visible to users with dark settings, so the style sheet was updated to fix that.

Addresses issue #13

johnb254 committed:
Adjust helpIcon styles

The help icon was not easily visible to users with dark settings, so the style sheet was updated to fix that.

Draxidious reviewed (changes requested):
The tooltip text box background no longer appears when hovering over a tool tip in dark mode, making the tooltip text hard to see. Also:
1. In light mode the icons should stay their normal black
2. In dark mode your changes to make them lighter should appear, with the tooltip textbox background unaltered

johnb254 committed:
Merge pull request #30 from wacker22/master

Update branch with changes from master

johnb254 committed:
Update styles.css

Adjusted colors to be more consistent. Changed style of links to better contrast background.
